core/templates/dev/head/domain/learner_dashboard/LearnerDashboardIconsDirective.js

```diff
@@ -34,10 +34,19 @@ oppia.directive('learnerDashboardIcons', [
         'LearnerDashboardActivityIdsObjectFactory',
         'LearnerPlaylistService',
         function(
-            $scope, LearnerDashboardIdsBackendApiService,
-            LearnerDashboardActivityIdsObjectFactory,
-            LearnerPlaylistService) {
+          $scope, LearnerDashboardIdsBackendApiService,
+          LearnerDashboardActivityIdsObjectFactory,
+          LearnerPlaylistService) {
           $scope.activityIsCurrentlyHoveredOver = true;
+          $scope.Flag = false;
+
+          $scope.$watch('Flag', function () {
+            console.log($scope.Flag);
+          });
+
+          $scope.enableToolTip = function (state) {
```

Comment:
- "Tooltip" is one word.
- Drop the space before the "(". Follow existing code conventions.
- "state" is a term of art in Oppia. Use a different name. (Better still, have separate enableTooltip and disableTooltip functions.)